fuse_i.h

```diff
@@ -391,25 +382,9 @@ struct fuse_conn {
 	/** rbtree of fuse_files waiting for poll events indexed by ph */
 	struct rb_root polled_files;
 
-	/** Maximum number of outstanding background requests */
-	unsigned max_background;
-
-	/** Number of background requests at which congestion starts */
-	unsigned congestion_threshold;
-
-	/** Number of requests currently in the background */
-	unsigned num_background;
```

Comment:
No need to remove this counter, helps with debugging.

Reply:
I don't see how and what particular problem this could possibly help to debug. There is nowhere for requests to go.